catalog-rest-service/src/main/resources/ui/src/pages/login/index.tsx

```diff
@@ -77,6 +77,9 @@ const SigninPage = () => {
                 Sign in with Github
               </button>
             )}
+            {appState.authProvider.provider === AuthTypes.AUTH0 && (
+              <button className="tw-signin-button">Sign in with Auth0</button>
+            )}
```

Comment:
@darth-coder00 we have too many conditions here.
I would suggest using a switch case that returns the elements instead of these conditions so JS doesn't need to validate every condition.

Reply:
Okay